NeoML/include/NeoML/TraditionalML/Svm.h

```diff
@@ -88,7 +90,12 @@ class NEOML_API CSvm : public ITrainingModel {
 
 private:
 	const CParams params; // classification parameters
-	CTextStream* log; // Logging stream
+	IThreadPool* const threadPool;
+	CTextStream* log = nullptr; // Logging stream
+
+	struct IThreadTask;
```

Comment:
Why are these 3 here? It looks like they can be removed from header.

Reply:
fixed